src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -2497,6 +2497,23 @@ else if (elem.equals("false")) {
         tagProcessor.registerStaticTag(MapTag.class, "parse_yaml", (attribute, object) -> {
             return (MapTag) CoreUtilities.objectToTagForm(YamlConfiguration.load(object.asString()).contents, attribute.context);
         });
+
+        // <--[tag]
+        // @attribute <ElementTag.millis_to_time>
+        // @returns TimeTag
+        // @group conversion
+        // @description
+        // Returns a TimeTag constructed from the number of milliseconds after the Unix Epoch (Jan. 1st 1970).
+        // See also <@link tag util.current_time_millis> and <@link tag TimeTag.epoch_millis>.
+        // @example
+        // - define time_now_millis <util.current_time_millis>

```

Comment:
This is not a good example - this is teaching a way to write a full script concept that isn't even a good way to write that